bitcoin/src/blockdata/block.rs

```diff
@@ -287,9 +287,9 @@ impl Block {
     }
 
     /// Returns the strippedsize of the block.
-    pub fn strippedsize(&self) -> usize {
-        let txs_size: usize = self.txdata.iter().map(Transaction::strippedsize).sum();
-        self.base_size() + txs_size
+    pub fn strippedsize(&self) -> Weight {
+        let txs_size: Weight = self.txdata.iter().map(Transaction::strippedsize).sum();
+        Weight::from_wu(self.base_size() as u64) + txs_size
```

Comment:
If we are going to do this why not make all the size functions return `Weight` as well? There is only virtual size and weight, right? When I look at the code I wonder "What does "size" as a `usize` mean"?